sim/dex.ts

```diff
@@ -826,6 +826,22 @@ export class ModdedDex {
 				ruleTable.set(ruleSpec, '');
 				continue;
 			}
+			if (ruleSpec.split(":").length > 1) {
+				const parts = ruleSpec.split(":");
+				if (parts[0] === 'teamlength') {
+					const vals = ruleSpec.split(":")[1].split('-');
+					const onValidate = parseInt(vals[1]);
+					const onBattle = parseInt(vals[0]);
+					format.teamLength = {
+						validate: [onBattle, onValidate],
+						battle: onBattle
+					}
+				}
+				else if (parts[0] === 'level') {
+					format.maxLevel = parseInt(parts[1]);
+				}
+				continue;
```

Comment:
The `continue` combined with the check above would eliminate `:` from being supported in any other rule name. This does happen after `+` and `-` so you're at least not filtering out actual legitimate bans, but I'm still not super comfortable with this.

Reply:
I figured any future rules using `:` would most likely end up in the same code block (before the continue) anyway as it doesn't make much sense to split by `:` twice